index.html

```diff
@@ -8,53 +8,17 @@
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
-<body>
-  <nav class="navbar navbar-nav">
-    <a href="#" class="logo"></a>
-    <a href="#"><span class="logout">Logout</span></a>
-  </nav>
-
-  <div class="list-container">
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-    <div class="list-item">
-      <a class="item-book-cover" href="detail.html"></a>
-      <span class="item-bold-text">Título del libro</span>
-      <span class="item-text">Autor del libro</span>
-    </div>
-  </div>
+<body class="login-container">
+  <form class="login-form" action="file:///Users/samirhernandotapierotovar/Frontend/st-angular/list.html">
+    <img class="form-logo" src="resources/logo-wolox.png" alt="logo-wolox"/>
+    <div class="form-content border-line-silver">
```

Comment:
change input elements of type submit and button in elements buttons because this _wrapper div_ is unnecessary, try to do the silver line as a pseudoelement that it comes out of some buttons.